cleanup_integration_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash +e
```

Comment:
Yeah, that was the intent, anyway -- just to make sure that it doesn't bail out if one of the cleanup scripts fails...now that I'm checking, though, I'm not sure that's actually correct, or if it just happens to work because that's what my default behavior is with MinGW.

 - [ ] Just use `set +e` like a normal person